waku/waku_archive/driver/builder.nim

```diff
@@ -78,22 +80,33 @@ proc new*(T: type ArchiveDriver,
     return ok(res.get())
 
   of "postgres":
-    const MaxNumConns = 5 #TODO: we may need to set that from app args (maybe?)
-    let res = PostgresDriver.new(url, MaxNumConns, onErrAction)
-    if res.isErr():
-      return err("failed to init postgres archive driver: " & res.error)
-
-    let driver = res.get()
-
-    try:
-      # The table should exist beforehand.
-      let newTableRes = waitFor driver.createMessageTable()
-      if newTableRes.isErr():
-        return err("error creating table: " & newTableRes.error)
-    except CatchableError:
-      return err("exception creating table: " & getCurrentExceptionMsg())
-
-    return ok(driver)
+    when defined(postgres):
+      const MaxNumConns = 5 #TODO: we may need to set that from app args (maybe?)
+      let res = PostgresDriver.new(url, MaxNumConns, onErrAction)
+      if res.isErr():
+        return err("failed to init postgres archive driver: " & res.error)
+
+      let driver = res.get()
+
+      try:
+        # The table should exist beforehand.
+        let newTableRes = waitFor driver.createMessageTable()
+        if newTableRes.isErr():
+          return err("error creating table: " & newTableRes.error)
+      except CatchableError:
+        return err("exception creating table: " & getCurrentExceptionMsg())
+
+      return ok(driver)
+
+    else:
+      ## Error variable to be used if the user is trying to mount Store with a Postgres database
+      var errPgMsg = "ERROR:"
+      errPgMsg &= "The POSTGRES flag should be passed to the make command, e.g. "
```

Comment:
It sounds me that the message is more like targeting developers not node operators.
It's lucky when it is met.
I would rather say something "This build does not support store with PostgresSQL."